ts/a11y/explorer.ts

```diff
@@ -246,8 +246,7 @@ export function ExplorerMathDocumentMixin<B extends MathDocumentConstructor<HTML
         ...BaseDocument.OPTIONS.renderActions,
         explorable: [STATE.EXPLORER]
       }),
-      a11y: expandable({
-        ...BaseDocument.OPTIONS.a11y,
+      a11y: {
```

Comment:
The `locale` and `speechRules` properties are already in the `BaseDocument.OPTIONS.a11y` since they are in the `EnrichedMathDocument` document class created by the `semantic-enrich` extension, that is the `BaseDocument` here (via `HTMLDOCUMENT`). So I think you want to keep the `...BaseDocument.OPTIONS.a11y` that was old line 250 and not include those two properties here.

Also, the `a11y` property in the `OPTIONS` in `semantic-enrich` should also be made `expandable()` so that if enrichment is enabled before the explorer, you don't get errors about these other explorer options.

Reply:
I see what you mean. But the more I think about it, the more I think all this should be done differently as to not conflate concerns. Here's an alternative proposal:
Let's have two set of options: `sre` and `a11y`.
* `sre` are defined expandable on both `EnrichedMathDocument` and `ExplorerMathDocument`. They are directly passed to SRE, which does its own sifting anyway, so no need for syntax checking or defaults (we might want to set `domain`, though). 
* `a11y` would be options only for the explorer, only defined on `ExplorerMathDocument`, non-expandable.

This should simplify the places where we call `setupEngine`. It would also allow us to make use of additional SRE features. E.g, at the moment enrichment can only use normal speech, but I could see a use case for enriching with SSML markup, which will need the `markup` parameter to be set. `enrichSpeech` would become an `sre.speech` option, no conflict with the `a11y.speech` counterpart. We also ensure that the SRE options in MathJax are the same as in SRE itself and thereby avoid confusion.

Does that make sense?

The only thing we need to be mindful about is the connection between `a11y.speechRules` and `sre.domain`, `sre.style`. But that should be doable.

Reply:
> `...`(we might want to set `domain`, though).

And `speech: 'none'` of course. I did not think that far when writing that line.

Reply:
I think the separation you describe is a good one.  Can you make the needed changes, or do you need me to do anything?

It does make a possible breaking change to the options, but I doubt many are setting these in their pages, so I'm not too worried about that.

Reply:
> I think the separation you describe is a good one. Can you make the needed changes, or do you need me to do anything?
>

I'll do that.

> It does make a possible breaking change to the options, but I doubt many are setting these in their pages, so I'm not too worried about that.

That is true. We can keep `a11y` expandable for this patch release and then make the breaking change for the next point release.

Reply:
OK, sounds good.

Reply:
OK, this looks pretty good.

Question:  Are there other SRE options that can appear in the `sre` block?  If not, then you don't need an `expandable()` for that.  If there are, could we give their values here, or are they too esoteric (or too many)?  As it stands, the `expandable()` will mean that no errors will be reported for misspelled or undefined options.  You can add to the options later as you do in `setA11yOptions()` even if they aren't in the initial `OPTIONS`.  You only need `expandable()` if the user can include arbitrary options, or if more options will be added by another handler later.  I thought the reason to separate `sre` from `a11y` was to avoid that situation.

Reply:
I'd rather keep it `expandable` for a few reasons:

* There are about a dozen options (despite the imminent removal of the `cache` option). Not all are exposed and some are rather esoteric, e.g., `pitch` and `rate` set baselines for systems using `ssml` or `acss` markup, like, mathlive, emacspeak, chromevox. They still might make sense server-side.
* Some options are simply ignored if not set, but can influence behaviour even if set to `0`, `null`, or `false`.
* We don't have to keep options in sync: E.g., if I were to use MathJax-full with a newer version of SRE and pass on a new option I can do that without MathJax complaining that there is no default.

Reply:
OK, just was checking to make sure it was OK that unknown options were not being flagged.